### DISCUSSION TITLE
Assign a discussion to a Project

### DISCUSSION THREAD

**Discussion by karnaker** [community/community#3775] [Discussions]
You should be able to assign a discussion to a Project.

This feature should work the same way as the Issues equivalent. On the right side of a discussion, you should be able to click "Projects" and select one or more project boards where you want to add the discussion.

Project boards help you organize and prioritize work, and discussions are an important part of work.

**Comment by ashleykolodziej** [comment]
I would like the ability to add a discussion to a project as well, so that I can create a roadmap using discussions directly, as well as see the status of discussions (answered, unanswered, triaged, and added to roadmap) easily without having to duplicate the information in another repo.

**Comment by bennypowers** [comment]
We'd like to track ideas that are not ready to be assigned as "units of work" - cards marked "needs discussion" that don't belong in our issues feed, but would eventually be graduated there. For that, we'd like to add discussions to our project board, just like we do issues or PRs

**Comment by sumau** [comment]
Duplicate of #11223

**Reply by gregsdennis** [reply]
Pedantically, that's a duplicate of this one.

**Comment by im-chabani** [comment]
adding my support to this, it owuld be great !

**Comment by Vimalraaj1512** [comment]
I would like to track ideas that are not ready to be assigned as a projects.

**Comment by shamoon** [comment]
Five years, please add this and make Projects more useful. Our project uses Discussions for all feature requests, so we cant track them at all in projects.
